content/references/javascript-library/customizing-your-registration-schema/index.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Customizing Your Registration Schema"
+tags: ["LoginRadiusV2.js", "JavaScript Library", "Registration Schema", "Registration Fields"]
+description: "This is a document detailing registration schema customizations."
+path: "/references/javascript-library/customizing-your-registration-schema"
+---
+
+# Customizing Your Registration Schema
+
+Your application follows a configured schema which populates the registration forms generated using the LoginRadius JavaScript Library (LoginRadiusV2.js). You are free to override this default configuration during the initialization of the JavaScript Library.
+
+## Schema Format
+
+A schema object is defined using the following format:
+
+```js
+{
+  "type": "",
+  "name": "",
+  "display": "",
+  "rules": ""
+}
+```
+
+  * **type**: The type of field to display. This key supports `string` or `password` as possible values. `string` will display the consumer's input normally. `password` will hide the consumer's input.
```

Comment:
Type supports "string", "password", "checkbox", "option", "hidden", "Email" and "text".

Reply:
What does option, hidden, email and text do? Etc. how are they different from string?

Reply:
"option" for Dropdown like Gender (Male/Female)
"email" if we pass it email them some browsers will handle the validation  for email.
"hidden" Not sure about the proper use case. but we can give customer for customization purpose or we can remove ti from this doc.